docs/.vuepress/config.js

```diff
@@ -1,141 +1,251 @@
 module.exports = {
-  theme: "cosmos",
-  title: "Kava Documentation",
-  themeConfig: {
-    logo: {
-      src: "/logo.svg",
+  "theme": "cosmos",
```

Comment:
Any reason for adding the quotes?
In general it's easier to see code changes when not combined with formatting changes.

Reply:
@rhuairahrighairidh I was using yaml to write the changes and taking the json output which is why you see the double quotes, I can revert it back to no quotes!